Misc/NEWS.d/next/Library/2025-03-30-19-55-10.gh-issue-131792.NNjzFA.rst

```diff
@@ -0,0 +1,5 @@
+Improved performance of :func:`textwrap.dedent` by an average of ~2.4x,
```

Comment:
Can we maybe also add a versionchanged for that?

Reply:
I assume you mean for the bug fix? (the comment is on the performance line)

How about:

```rst
.. versionchanged:: next
   The :func:`!dedent` function now correctly normalizes blank lines containing
   only whitespace characters. Previously, the implementation only normalised
   blank lines containing tabs and spaces.
```

Reply:
Yes, something like this. Because it's still a behavioral change. Considering that dedent() is used probably quite frequently, a What's New is probably preferrable

Reply:
What section would you propsose for a _What's New_ entry?

Reply:
See #131924